Makefile

```diff
@@ -0,0 +1,14 @@
+SRC_FILES := $(shell find geo_to_hca -name \*.py -print)
+
+.PHONY: setup
+setup:
+	pip install -r requirements-dev.txt
```

Comment:
Please transition python projects to using pip-compile and pip-sync from [pip-tools](https://github.com/jazzband/pip-tools).
[This commit of the data archiver](https://github.com/ebi-ait/ingest-data-archiver/commit/d1bfd249976a6cad9e13a789337f526851cc7e94) might be a good example of transitioning from one to the other.

I can explain on call if needed.

Reply:
will do. Good idea.

Reply:
I migrated to pip tools. Can you please review?